src/analytics.js

```diff
@@ -129,9 +138,14 @@ Analytics.writeData = async function () {
 	}
 
 	for (const [key, value] of Object.entries(counters)) {
-		dbQueue.push(db.sortedSetIncrBy(`analytics:${key}`, value, today.getTime()));
+		incrByBulk.push([`analytics:${key}`, value, today.getTime()]);
 		delete counters[key];
 	}
+
+	if (incrByBulk.length) {
+		dbQueue.push(sortedSetIncrByBulk(incrByBulk));
```

Comment:
since there is only one element in dbQueue with this change, you can just move this down into the try catch block and use 

```
try {
  await sortedSetIncrByBulk(incrByBulk);
} catch (err) {
```
```